optimum/exporters/openvino/model_configs.py

```diff
@@ -2259,3 +2290,218 @@ def patch_model_for_export(
         if self._behavior == Phi3VisionConfigBehavior.VISION_EMBEDDINGS:
             return Phi3VisionImageEmbeddingsPatcher(self, model, model_kwargs)
         return super().patch_model_for_export(model, model_kwargs)
+
+
+class DummyQwen2VLLMInputGenerator(DummyTextInputGenerator):
+    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+        generated_input = super().generate(input_name, framework, int_dtype, float_dtype)
+        if input_name == "position_ids":
+            return generated_input.unsqueeze(0).expand(3, -1, -1)
+        return generated_input
+
+
+class DummyQwen2VLVisionEMbedInputGenerator(DummyVisionInputGenerator):
+    SUPPORTED_INPUT_NAMES = ("hidden_states",)
+
+    def __init__(
+        self,
+        task: str,
+        normalized_config: NormalizedVisionConfig,
+        batch_size: int = 1,
+        num_channels: int = DEFAULT_DUMMY_SHAPES["num_channels"],
+        width: int = 420,
+        height: int = 420,
+        **kwargs,
+    ):
+        self.batch_size = batch_size
+        self.height = height
+        self.width = width
+        self.num_channels = num_channels
+        self.temporal_patch_size = normalized_config.config.temporal_patch_size
+        self.patch_size = normalized_config.config.patch_size
+
+    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+        grid_h, grid_w = self.height // self.patch_size, self.width // self.patch_size
+        grid_t = self.batch_size
+        shape = [
+            grid_t * grid_h * grid_w,
+            self.num_channels * self.temporal_patch_size * self.patch_size * self.patch_size,
+        ]
+        return self.random_float_tensor(shape, framework=framework, dtype=float_dtype)
```

Comment:
why do we need `DummyQwen2VLVisionEmbedInputGenerator` ? We can just have `DummyQwen2VLVisionEmbedMergerInputGenerator` instead (`embed_dim` / `num_heads` attributes can be set tot None when not present in the config as not mandatory for `hidden_states`)
```suggestion
```

Reply:
not sure that I understand what you mean..

Reply:
I'm suggesting to only have one `DummyQwen2VLVisionEmbedInputGenerator` which would allow to generate to dummy inputs for both cases (all attributes that needed for all inputs generation are already defined in one which is currently  `DummyQwen2VLVisionEmbedMergerInputGenerator`, if not then some attribute might be removed in `DummyQwen2VLVisionEmbedMergerInputGenerator` as it appears some configs values won't be used in this case, I think we can use only one in both cases making some values optional (can be absent from the configs depending on what needs to be generated). I'm suggesting to have only one  `DummyQwen2VLVisionEmbedInputGenerator`, for which the behavior will impact the `hidden_states` shape (-> could depend on the input names which might be updated depending on the behavior, let me know what you think!)

Reply:
these models share the same config, so all values always present, the problem that both of them has input hidden_states and they have different sizes if we will use he same class we can not identify which one format should be used at specific moment

Reply:
removed, please take a look @echarlaix